src/tests/unit/services/VerifyAccountRequestProcessor.test.ts

```diff
@@ -251,5 +251,32 @@ describe("VerifyAccountRequestProcessor", () => {
 				verifyAccountRequestProcessorTest.calculateCopCheckResult({ ...hmrcVerifyResponse, nameMatches, accountExists }),
 			).toBe(result);
 		});
+
+		it("calls savePartialNameInfo if CopCheckResults is PARTIAL_MATCH", async () => {
```

Comment:
calls savePartialNameInfo if CopCheckResults is error?

Reply:
Left over copy and paste, removed test